lib/rubocop/cop/rspec/subject_declaration.rb

```diff
@@ -25,7 +25,7 @@ class SubjectDeclaration < Base
 
         # @!method offensive_subject_declaration?(node)
         def_node_matcher :offensive_subject_declaration?, <<~PATTERN
-          (send nil? ${#Subjects.all #Helpers.all} {(sym :subject) (str "subject")} ...)
+          (send nil? ${#Subjects.all #Helpers.all} {(sym #Subjects.all) (str #Subjects.all.map(&:to_s))} ...)
```

Comment:
```suggestion
          (send nil? ${#Subjects.all #Helpers.all} {(sym #Subjects.all) (str #Subjects.all)} ...)
```

Surprisingly, it works like this, too, at least specs do pass.

Reply:
Sorry for the late reply. Thanks for the correction.